test_runner/fixtures/benchmark_fixture.py

```diff
@@ -11,23 +11,28 @@
 from pathlib import Path
 
 # Type-related stuff
-from typing import Iterator, Optional
+from typing import TYPE_CHECKING, Callable, ClassVar, Iterator, Optional
 
 import pytest
-from _pytest.config import Config
-from _pytest.terminal import TerminalReporter
 from fixtures.types import TenantId, TimelineId
 
+if TYPE_CHECKING:
+    from _pytest.config import Config
+    from _pytest.config.argparsing import Parser
+    from _pytest.terminal import TerminalReporter
+    from fixtures.neon_fixtures import NeonPageserver
```

Comment:
Are you talking about the `NeonPageserver` or all things under `TYPE_CHECKING`? It's not necessary to include and execute the whole module if it's not used in the specific file

Reply:
I don't think so. It's possible to make `mypy` stricter, and it will alert before trying to push without typings.

Reply:
Think of it this way: let's say you're a C coder who sees this for the 1st time. Why would you care? Ok, it takes a bit more time to load an excessive module, so what? Not having to think about boilerplate like this is a big plus.

Reply:
As a C coder, you may face circular dependencies one day because of importing everything into everything. And it will be much harder to fix than learn about `typing.TYPE_CHECKING`. In my opinion, if you start to contribute to a different language, you should follow the best practices of this language. Btw, one day I will change the behavior with `global` variables in pytest because of this